test/integration/newapp_test.go

```diff
@@ -1711,7 +1712,7 @@ func TestNewAppSourceAuthRequired(t *testing.T) {
 	}
 }
 
-func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) string {
+func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) (string, func()) {
 	// Create test directories
 	testDir, err := ioutil.TempDir("", "gitauth")
```

Comment:
I'm trying to reduce an amount of garbage that is left on my file system after running the tests. And what you're suggesting, it doesn't give it to me -- these scripts won't remove these temporary files. Also I want to have ability to run unit tests from my IDE and don't rely on our scripts at all.